stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
…and also I think we set SWIFT_SOURCE_DIR/cmake/modules/ to be an include path, so you should just be able to say `include(StandaloneOverlay)`